### PR TITLE
Add blog post announcement for the CI/CD integration assistant

### PR DESCRIPTION
**Pull request by praneetloke**
### Proposed changes

Just as the title states...

### Unreleased product version (optional)

N/A

### Related issues (optional)

N/A

### CODE REVIEW

File: content/blog/announcing-ci-cd-integration-assistant/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Announcing CI/CD Integration Assistant for teams"
```
Comment (spara) [inline content/blog/announcing-ci-cd-integration-assistant/index.md:2]:
```suggestion
title: "Announcing CI/CD Integration Assistant for Teams"
```

File: content/blog/announcing-ci-cd-integration-assistant/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+authors: ["praneet-loke"]
+tags: ["CI/CD"]
+date: "2020-12-08"
+meta_desc: "We are excited to announce the launch of the integration assistant to make integration with CI/CD easy."
+meta_image: integration-assistant.png
+---
+
+We are excited to announce the launch of the integration assistant to make integration with CI/CD easy.
+Now you can easily get started with popular CI/CD services without the hassle of the initial setup.
```
Comment (spara) [inline content/blog/announcing-ci-cd-integration-assistant/index.md:11]:
```suggestion
Now, you can quickly get started with popular CI/CD services without the hassle of the initial setup.
```

File: content/blog/announcing-ci-cd-integration-assistant/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+<!--more-->
+
+We often hear developers wanting to setup CI/CD automation for their stacks but don't have the time or simply don't like
```
Comment (spara) [inline content/blog/announcing-ci-cd-integration-assistant/index.md:15]:
```suggestion
We often hear developers wanting to setup CI/CD automation for their stacks but don't have the time or don't like
```

File: content/blog/announcing-ci-cd-integration-assistant/index.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+the experience of figuring out how to configure a working pipeline. The integration assistant not only guides you with
```
Comment (spara) [inline content/blog/announcing-ci-cd-integration-assistant/index.md:16]:
```suggestion
the experience of figuring out how to configure a working pipeline. The integration assistant guides you with
```

File: content/blog/announcing-ci-cd-integration-assistant/index.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+instructions based on the CI/CD service, but also provides the customized pipeline configuration
```
Comment (spara) [inline content/blog/announcing-ci-cd-integration-assistant/index.md:17]:
```suggestion
instructions based on the CI/CD service and provides a customized pipeline configuration
```

File: content/blog/announcing-ci-cd-integration-assistant/index.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+* [GitLab CI/CD](https://docs.gitlab.com/ce/ci/)
+
+> You can always [request](https://github.com/pulumi/ci-workflow-templates/issues/new/choose) support for a new CI/CD service.
+
+### What is a VCS?
+
+A VCS is a version control system. You may already be familiar with Git-based services such as
+[Atlassian Bitbucket](https://bitbucket.org), [GitHub](https://github.com), [GitLab](https://gitlab.com).
+Adding your code to a central VCS allows you to secure your codebase against failures, as well as team collaboration.
+When you add an automated CI/CD pipeline your team will be able to deliver software safely and efficiently.
```
Comment (spara) [inline content/blog/announcing-ci-cd-integration-assistant/index.md:34]:
```suggestion
When you add an automated CI/CD pipeline, your team can deliver software safely and efficiently.
```

File: content/blog/announcing-ci-cd-integration-assistant/index.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+The assistant also goes beyond helping you get started with an automated pipeline. It works throughout the Pulumi Console
+providing hints for configuring a VCS for projects that don't have one, as well as identifying updates in the activity
```
Comment (spara) [inline content/blog/announcing-ci-cd-integration-assistant/index.md:37]:
```suggestion
providing hints for configuring a VCS for projects that don't have one and identifying updates in your stack's activity
```

File: content/blog/announcing-ci-cd-integration-assistant/index.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+tab of you stack that were not run in an automated pipeline. These additions will help you quickly identify the stacks
```
Comment (spara) [inline content/blog/announcing-ci-cd-integration-assistant/index.md:38]:
```suggestion
tab that were not run in an automated pipeline. These additions will help you quickly identify the stacks
```

File: content/blog/announcing-ci-cd-integration-assistant/index.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+where the best practices are not being followed and take corrective actions.
```
Comment (spara) [inline content/blog/announcing-ci-cd-integration-assistant/index.md:39]:
```suggestion
where the best practices are not followed and take corrective actions.
```

File: content/blog/announcing-ci-cd-integration-assistant/index.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+Access the wizard in the Integrations option of your stack's settings page to help you through the whole process of
+configuring a VCS to setting up a working automated pipeline.
+
+Learn more about the [CI/CD integration assistant]({{< relref "/docs/intro/console/extensions/ci-cd-integration-assistant" >}}).
+
+We are eager to hear your feedback on how we can improve your experience with this feature. If you
```
Comment (spara) [inline content/blog/announcing-ci-cd-integration-assistant/index.md:46]:
```suggestion
We are eager to hear your feedback to improve your experience with this feature. If you
```

**Review by spara — CHANGES_REQUESTED** [review]
Minor edits to improve readability

**Comment by praneetloke** [comment]
@spara thank you for the great suggestions as always! I have applied your suggestionss.

**Pull request closed without merging**